pyseq.py

```diff
@@ -80,6 +83,14 @@
 warnings.simplefilter('always', DeprecationWarning)
 
 
+def _natural_key(x):
```

Comment:
basically sorts files into the order they should be in to get added to a sequence